sass/style.css

```diff
@@ -181,12 +220,34 @@ h3.city {
   text-transform: uppercase;
 }
 
+.locations-container {
+  padding: 0 !important;
```

Comment:
You can remove the px-2 and m-2 classes from the element, and eliminate the need for !important.